package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "import-export-app",
-    "version": "1.5.69",
+    "version": "99.9.9",
```

Comment:
I guess this is so that the first release will be 100.0.0?

Reply:
yeah exactly - for context, here is a writeup of the steps needed to move to continuous release: https://github.com/dhis2/notes/discussions/293